utils/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -81,13 +81,20 @@ object OkHttpClientHelper {
         return OkHttpClient.Builder()
             .addNetworkInterceptor { chain ->
                 val request = chain.request()
-                chain.proceed(
-                    if (request.header("User-Agent") == null) {
-                        request.newBuilder().header("User-Agent", Environment.ORT_USER_AGENT).build()
-                    } else {
-                        request
-                    }
-                )
+                @Suppress("TooGenericExceptionCaught")
+                try {
+                    chain.proceed(
+                        if (request.header("User-Agent") == null) {
+                            request.newBuilder().header("User-Agent", Environment.ORT_USER_AGENT).build()
+                        } else {
+                            request
+                        }
+                    )
+                } catch (e: Exception) {
+                    log.error { "HTTP request to '${request.url}' failed with exception." }
```

Comment:
Better:
`logger.error(e) {"HTTP request to '${request.url}' failed with exception." }`

Then you don't need the `showStackTrace`.

Reply:
Should be `log` instead of `logger`, but nice otherwise, I didn't know about this overload.

Reply:
`showStackTrace` takes the command line option into account whether stack traces should be printed. I think, it is therefore common to use it.